packages/as-types/assembly/argument.ts

```diff
@@ -290,6 +302,18 @@ export class Args {
     }
     return this;
   }
+
+  /**
+   * Adds an object to the serialized byte array if the given object implements `Serializable`
+   *
+   * @see {@link Serializable}
+   * @param object - object to add
+   * @returns the modified Arg instance
+   */
+  addObject<T extends Serializable<T>>(object: T): Args {
```

Comment:
you can add it to add. How ?

```typescript
...
    } else if (arg instanceof f64) {
      this.serialized = this.serialized.concat(f64ToBytes(arg as f64));
    } else if (arg instanceof Serializable) {
      this.serialized = this.serialized.concat(arg.serialize());
    } else {
      ERROR('args doesn't know how to serialize the given type.')
    }
```
the else is free ;-). But should be tested.

Reply:
I'll use ts-ignore